contracts/EpochTokenLocker.sol

```diff
@@ -36,7 +36,7 @@ contract EpochTokenLocker {
     mapping(address => mapping(address => BalanceState)) private balanceStates;
 
     // user => token => batchId => wasCredited
```

Comment:
I would think this comment should be adjusted to coincide with the changes to `lastCreditBatchId`.